src/pages/coaches/richard-moss.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Richard Moss
+title: Senior React Engineer
+companyName: LeanJS
+companyLink: https://leanjs.com
+gitHub: https://github.com/ric9176
+medium: https://medium.com/@richard_13764
+twitter: https://twitter.com/ric9176
+linkedIn: https://uk.linkedin.com/in/richard-moss-55881b82
+blockquote:
+  Collaboratively working with developers to help them understand concepts such as the functional programming
+  paradigm helps them take their skills to the next level - and I learn a lot too!
+imageDescription: React GraphQL Academy coach Richard Moss, looking pretty happy with a beach in the background
+imageSrc: https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/team%2Frichard.jpg?alt=media
+tags: ['beginner']
+---
+
+<ul>
```

Comment:
@ric9176 why don't we use markdown for the list? https://www.markdownguide.org/basic-syntax/#unordered-lists